lib/resources/r4/related_person_patch.yaml

```diff
@@ -131,6 +142,44 @@ operations:
         <li>Requires a test operation to be provided for the `Relationship.id` field of the relationship to be removed.</li>
       </ul>
 
+  - name: replace-relationship-extensions
+    path: /relationship/{index}/extension
+    operation: replace
+    type: string
+    url: https://hl7.org/fhir/r4/related-person-definitions.html#RelatedPerson.relationship
+    description: Replace the `period` and `relation` of the relationship at the given <code>{index}</code>.
```

Comment:
Needs clarification, should we name them as extensions?

Reply:
I am fine with naming them as extensions if you think it would make it more clear.

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/662/commits/d55126717a21ef75564d0c386dd49a8801793c8f#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR148